tests/e2e-openshift/must-gather/chainsaw-test.yaml

```diff
@@ -0,0 +1,105 @@
+apiVersion: chainsaw.kyverno.io/v1alpha1
+kind: Test
+metadata:
+  name: must-gather
+spec:
+  namespace: chainsaw-must-gather
+  steps:
+  - name: Install Target Allocator
+    try:
+    - apply:
+        template: true
+        file: install-target-allocator.yaml
+    - assert:
+        file: install-target-allocator-assert.yaml
+    catch:
+    - podLogs:
+        selector: app.kubernetes.io/component=opentelemetry-targetallocator
+  - name: assert-data-propagates
```

Comment:
Why are we asserting this separately here, as opposed to putting it in `install-target-allocator-assert.yaml`?